src/A1_csv_to_json.py

```diff
@@ -328,27 +328,28 @@ def create_json_from_csv(input_directory, filename, parameters, storage=False):
         )
 
     # check parameters
-    extra = list(set(parameters) ^ set(df.index))
-    if len(extra) > 0:
-        for i in extra:
-            if i in parameters:
-                logging.error(
-                    "In the file %s.csv" % filename
-                    + " the parameter "
-                    + str(i)
-                    + " is missing. "
-                    "check %s",
-                    input_directory + " for correct parameter names.",
-                )
-            else:
-                logging.error(
-                    "In the file %s.csv" % filename
-                    + " the parameter "
-                    + str(i)
-                    + " is not recognized. \n"
-                    "check %s",
-                    input_directory + " for correct parameter names.",
-                )
+    if storage is False:
```

Comment:
> if storage is False:

Can you change  `storage` to a variable name that is directly graspable, eg. `asset_is_a_storage` ?